### DISCUSSION TITLE
Update 20210731 has broken main screen display.

### DISCUSSION THREAD

**Discussion by Roland-B** [rern/rAudio#455] [Fixed Issues] [answered]
@rern n - update has broken main screen?! Icon missing from browser full screen, playing time and icon missing from mobile/tablet screen as per below.

![2021-07-31_104102](https://user-images.githubusercontent.com/43496391/127735957-632201a4-88a1-448d-b29c-3fed88fe8433.jpg)

![signal-2021-07-31-103552](https://user-images.githubusercontent.com/43496391/127735960-bdbb380c-295b-4699-b622-7febc4f94984.jpeg)

RolandB.

rAudio 1 • 20210731
5.10.52-1-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Comment by Roland-B** [comment, marked as answer]
@rern Thanks for the improvement, but still not fully fixed. Still no MPD Icon displayed.

RolandB

rAudio 1 • 20210807
5.10.52-1-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Reply by rern** [reply]
There's been no MPD at mode icon area. No icons implies file mode.

**Reply by Roland-B** [reply]
@rern OK, my mistake. Thanks for confirming.

Fixed with update 20210807.
